nuxeo-compound-documents/src/test/java/org/nuxeo/compound/documents/TestCompoundDocumentUtils.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * (C) Copyright 2022 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package org.nuxeo.compound.documents;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.nuxeo.compound.documents.CompoundDocumentConstants.COMPOUND_DOCTYPE;
+import static org.nuxeo.compound.documents.CompoundDocumentConstants.COMPOUND_FOLDER_DOCTYPE;
+
+import java.io.File;
+import java.io.IOException;
+
+import org.nuxeo.common.utils.ZipUtils;
+import org.nuxeo.ecm.core.api.Blob;
+import org.nuxeo.ecm.core.api.Blobs;
+import org.nuxeo.ecm.core.api.CoreSession;
+import org.nuxeo.ecm.core.api.DocumentModel;
+import org.nuxeo.ecm.core.api.PathRef;
+import org.nuxeo.runtime.api.Framework;
+
+/** @since 2021.0 */
+public final class TestCompoundDocumentUtils {
+
+    private TestCompoundDocumentUtils() {
+        // Utility class
+    }
+
+    protected static Blob makeTestZipBlob() throws IOException {
+        File[] source = { new File(TestCompoundDocumentImporter.class.getResource("/files/a").getPath()),
+                new File(TestCompoundDocumentImporter.class.getResource("/files/a.txt").getPath()) };
+        File newZip = Framework.createTempFile("test", ".zip");
+        ZipUtils.zip(source, newZip);
+        Blob blob = Blobs.createBlob(newZip);
```

Comment:
Can't we get rid of the temp file by using a simple OutputStream? And moreover do we want it or are we good with the temp file? 😬 
@kevinleturc ?

Reply:
I think we're good with the temp file, it is unique and stored under the target/nxruntime directory.

Using OutputStream could be more complex as we will need to convert it to an InputStream in order to create the blob (which in the end create a temp file).